README.md

```diff
@@ -283,20 +290,19 @@ sudo -E python setup.py install build_ext --openssl=/usr/local/opt/openssl/
 #### Optional Requirements
 
 If you want to support revocation, you also need to have cfssl installed and in your
-path on the tenant agent.  It can be obtained from https://github.com/cloudflare/cfssl.  You
-will also need to set ca_implementation to "cfssl" instead of "openssl" in `/etc/keylime.conf`.
+path on the tenant agent.  It can be obtained from [cloudfare/cfssl](https://github.com/cloudflare/cfssl). You will also need to set ca_implementation to "cfssl" instead of "openssl" in `/etc/keylime.conf`.
```

Comment:
This reads kind of strange. When rendered, although "cloudflare/cfssl" is a link, the sentence doesn't actually make sense since "can be obtained from X" means X is usually a location and "cloudflare/cfssl" is not a location but it's name.

Reply:
Yes as you pointed it out, I think so too, i'll revert this back to it's original form. Thanks for all the suggestions!